.travis/git_utils.py

```diff
@@ -14,16 +14,15 @@ def git(*args):
         sys.exit(err.returncode)
 
 
-# Root of the Git repository
-ROOT = git("rev-parse", "--show-toplevel")
-
-
-def get_changed_paths(*args):
+def get_changed_paths(*args, globs=[]):
```

Comment:
Beware! Using mutable default arguments in Python is a bad habit, because any mutation within the body of the function persists between calls. Example:

```pylon
>>> def print_names(names=[]):
...     names.append("alex")
...     print(names)
...
>>> print_names(names=["jamie", "harrison"])
['jamie', 'harrison', 'alex']

>>> print_names()
['alex']

>>> print_names()
['alex', 'Alex']

>>> print_names()
['alex', 'alex', 'Alex']
```

Preferred:

```python
def get_changed_paths(*args, globs=None):
    if globs is None:
        globs = []
```

Reply:
Actually, you don't even need the default argument here, because you've fixed the two places where this function is called.